src/windows/views/webview_backend/webkit.py

```diff
@@ -123,3 +123,20 @@ def keyPressEvent(self, event):
         else:
             # Ignore most keypresses
             event.ignore()
+
+    def wheelEvent(self, event):
+        """ Mousewheel scrolling """
+        if event.modifiers() & Qt.ShiftModifier:
+            event.accept()
+            frame = self.page().mainFrame()
+            # Compute scroll offset from wheel motion
+            tick_scale = 120
+            steps = int(event.angleDelta().y() / tick_scale)
+            delta = -(steps * 100)
+            log.debug("Scrolling horizontally by %d pixels", delta)
+            # Update the scroll position using AngularJS
+            js = f"$('body').scope().scrollLeft({delta});"
```

Comment:
I think the build failure is my fault. I used an f-string, forgetting that we're still testing on Xenial which is running Python 3.5 (despite it being EOL since last spring). \*sigh\* Until we can get rid of Xenial when _it_ reaches EOL in April, I guess this will have to be...
```suggestion
            js = "$('body').scope().scrollLeft({});".format(delta)
```

Reply:
I'm just going to apply that change directly, to kick off a new build and hopefully wrap this up.

Reply:
Yup, that was it. Now I'm really frustrated that the error message there wasn't clearer. I guess we should _log_ the exception from the failed WebKit backend load, before raising the `RuntimeError` from the WebEngine exception. Otherwise the reason for the WebKit import failure gets lost.

Well, I'll fix that in another PR. For this change, merging! Many thanks @MatthieuMeert!